src/main/webapp/less/awesome-bootstrap-checkboxcss.less

```diff
@@ -0,0 +1,317 @@
+.checkbox {
```

Comment:
could this be added to the `resource-server` then imported into Feedback portlet?